.env-example

```diff
@@ -1,5 +1,9 @@
 # Example ENV VARIABLES
-REDIS_URL=redis://redis:623e9c0cb0a185b9cac9b3c8geal3812@joe.redis.com:12345/
-ACCESS_CONTROL_ALLOW_ORIGIN=*
-AWS_ACCESS_KEY_ID=ABCDEFGHIJKLMNOPQRST
-AWS_SECRET_ACCESS_KEY=A1b2C3D4/e5-f6g7j2K2x2i13n1+0x21xp7Ux-sl
\ No newline at end of file
+REDIS_URL="redis://redis:623e9c0cb0a185b9cac9b3c8geal3812@joe.redis.com:12345/"
+AWS_ACCESS_KEY_ID="ABCDEFGHIJKLMNOPQRST"
```

Comment:
:)
is it the same case for the redis usernaame & password? and the keys in the history?